### PR TITLE
bclm, autoraise: add page; gallery-dl: edit page

### LINKED ISSUE #14996: Page modification request: gallery-dl
### Command description

Linked my update so it can be used:

https://github.com/plshelpidkwhatimdoing/pages/blob/main/gallery-dl.md

### Command details

_No response_

### Documentation

https://gdl-org.github.io/docs/

### Platform

Common

### VCS repository link (e.g. GitHub, GitLab)

https://github.com/mikf/gallery-dl

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Laaarge**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**0.1.0 ; v5.3.;  

Closes #14996.

### CODE REVIEW

File: pages/osx/autoraise.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+> More information: <https://github.com/sbmpost/AutoRaise>.
+
+- Compile the source code:
+
+`make clean && make`
+
+- Alternative options are available:
+
+`make clean && make CXXFLAGS="-DOLD_ACTIVATION_METHOD -DEXPERIMENTAL_FOCUS_FIRST"`
+
```
Comment (acuteenvy) [inline pages/osx/autoraise.md:15]: We don't document installation methods.
```suggestion
```

File: pages/osx/bclm.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# bclm
+
+> Set a custom chrage limit on MacBooks.
+> More information: <https://github.com/zackelia/bclme>.
+
+- Install the wrapper:
+
+`brew tap zackelia/formulae && brew install bclm`
+
```
Comment (acuteenvy) [inline pages/osx/bclm.md:9]:
```suggestion
```

File: pages/common/gallery-dl.md [hunk 3]
```diff
@@ -7,6 +7,10 @@
 
 `gallery-dl "{{url}}"`
 
+- Save images to a specific directory/{{relevant_folder}}, use --directory to avoid that:
```
Comment (gutjuri) [inline pages/common/gallery-dl.md:10]:
```suggestion
- Save images to a specific directory:
```

Not sure what you meant by the last half-sentence. Also, I removed the placeholder in the description.

File: pages/common/gallery-dl.md [hunk 4]
```diff
@@ -7,6 +7,10 @@
...
+
+`gallery-dl --destination "{{path/to/directory}}" "{{url}}"`
```
Comment (gutjuri) [inline pages/common/gallery-dl.md:12]:
```suggestion
`gallery-dl --destination {{path/to/directory}} "{{url}}"`
```

Usually, we don't put local paths in quotes

File: pages/osx/autoraise.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# AutoRaise
+
+> Adds Focus Follows Mouse to Macos.
```
Comment (gutjuri) [inline pages/osx/autoraise.md:3]:
```suggestion
> Add Focus-Follows-Mouse to MacOS.
```

File: pages/osx/autoraise.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+> Generates binary & app. standard options are available as standalone release.
+> Add the binary or app to login item to start on boot.
```
Comment (gutjuri) [inline pages/osx/autoraise.md:5]:
```suggestion
```

I'd remove this, as this seems to be a description of installation methods

File: pages/osx/autoraise.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+> More information: <https://github.com/sbmpost/AutoRaise>.
+
+- Default setup is documented in the readme, and shown on execution (the \& runs the process in the background, not necessary):
```
Comment (gutjuri) [inline pages/osx/autoraise.md:8]:
```suggestion
-  Run AutoRaise in the background:
```

File: pages/osx/bclm.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# bclm
+
+> Set a custom chrage limit on MacBooks.
```
Comment (gutjuri) [inline pages/osx/bclm.md:3]:
```suggestion
> Set a custom charge limit on MacBooks.
```

File: pages/osx/bclm.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://github.com/zackelia/bclme>.
+
+- Set the charge limit to 80% (battery charge level may be slightly lower than set value for intel machines):
```
Comment (gutjuri) [inline pages/osx/bclm.md:6]:
```suggestion
- Set the charge limit to 80% (battery charge level may be slightly lower than set value for Intel machines):
```

File: pages/osx/bclm.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+`sudo bclm write 77`
```
Comment (gutjuri) [inline pages/osx/bclm.md:8]:
```suggestion
`sudo bclm write {{77}}`
```

Why is it 77 when the description says 80%?

File: pages/osx/bclm.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+- Read the current charge limit:
+
+`bclm read`
+
+- Keep the charge limit after reboot/smc reset:
```
Comment (gutjuri) [inline pages/osx/bclm.md:14]:
```suggestion
- Keep the charge limit after rebooting/smc reset:
```

File: pages/osx/bclm.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+
+`sudo bclm persist`
+
+- Remove the persisting charge limit:
```
Comment (gutjuri) [inline pages/osx/bclm.md:18]:
```suggestion
- Remove the persistent charge limit:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thank you very much! I have added some suggestions and questions.

File: pages/osx/bclm.md (hunk 10 above)
Comment (Laaarge) [reply pages/osx/bclm.md:8]: It is in order to show that the battery level is not always exactly the same as set limit.
I thought it was useful to show, but it is maybe not strictly necessary ?

**Review by Laaarge — COMMENTED** [review]
changes done

File: pages/osx/bclm.md (hunk 10 above)
Comment (acuteenvy) [reply pages/osx/bclm.md]: > It is in order to show that the battery level is not always exactly the same as set limit. I thought it was useful to show, but it is maybe not strictly necessary ?

Could you explain that? If the description says that the command sets the limit to 80%, then it should do exactly what the description tells you it does. If you think more information is needed, feel free to add something to the description in a way that's not misleading.

Comment (Laaarge) [reply pages/osx/bclm.md]: the doc of the utility states the following:
> For Intel machines, when writing values, macOS charges slightly beyond the set value (~3%). In order to display 80% when fully charged, it is recommended to set the BCLM value to 77%. When charging while system is shut down or sleeping, the charging can go beyond set value more than average 3%.
and
> For Apple silicon machines, only the values 80 and 100 are supported and firmware >= 13.0 is required.

are those things that sould be understanble from the tldr page, or do we consider that people are going to go through the link if it seems that something is off ?

File: pages/osx/bclm.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+# bclm
+
+> Set a custom charge limit on MacBooks.
+> More information: <https://github.com/zackelia/bclme>.
+
+- Set the charge limit to 80% (battery charge level may be slightly lower than set value for Intel machines):
```
Comment (acuteenvy) [inline pages/osx/bclm.md:6]:
```suggestion
- Set the charge limit to about 80% (for Intel machines, the battery charge level may be slightly lower than the set value):
```

File: pages/osx/autoraise.md [hunk 14]
```diff
@@ -0,0 +1,8 @@
+# AutoRaise
+
+> Add Focus-Follows-Mouse to MacOS.
```
Comment (acuteenvy) [inline pages/osx/autoraise.md:3]: From https://github.com/sbmpost/AutoRaise.

The previous description is kind of hard to understand in my opinion.
```suggestion
> Automatically raise and focus on a window when hovering over it with the mouse.
```

**Review by acuteenvy — APPROVED** [review]

File: pages/osx/bclm.md (hunk 10 above)
Comment (acuteenvy) [reply pages/osx/bclm.md]: Sounds fine, thanks.

**Review by gutjuri — APPROVED** [review]
Thanks for the changes!

**Review by karthik-script — APPROVED** [review]

**Pull request merged**